packages/design-system-react-native/src/components/AvatarBase/AvatarBase.test.tsx

```diff
@@ -524,13 +524,13 @@ describe('AvatarBase', () => {
       'rounded-full',
       TWCLASSMAP_AVATARBASE_SIZE_DIMENSION[AvatarBaseSize.Md],
     ].join(' ');
-    const customClasses = `${baseClasses} extra-class`;
+    const customClasses = `${baseClasses} bg-default`;
```

Comment:
Because we are enforcing no non-tailwind classnames we have to update tests with mock clasnames to valid tailwind classnames